packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -15,38 +15,38 @@
 
   display: inline-block;
   position: relative;
-  margin-block: var(--SpaceMarginBlock);
-  margin-inline: var(--SpaceMarginInline);
-  padding-block: var(--SpacePaddingBlock);
-  padding-inline: var(--SpacePaddingInline);
+  margin-block: var(--MarginBlock);
+  margin-inline: var(--MarginInline);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
   transition-duration: var(--TransitionDuration);
   transition-timing-function: var(--TransitionTiming);
   border-width: var(--BorderWidth);
   border-style: var(--BorderStyle);
-  border-color: var(--ColorBorderPrimary);
-  background-color: var(--ColorBackgroundPrimary);
-  color: var(--ColorText);
+  border-color: var(--PrimaryBorderColor);
+  background-color: var(--PrimaryBackgroundColor);
+  color: var(--TextColor);
```

Comment:
Based on my understanding of the writeup this should just be `Color` to match the CSS Declaration syntax

Reply:
@andrewberg-okta's example in the document uses `TextColor`, so I've used it throughout my renaming so far. Happy to remove if we can get a confirmation.

Reply:
@edburyenegren-okta yes thank you! I just went back and double checked. [Asked the clarifying question here](https://oktawiki.atlassian.net/wiki/spaces/UX/pages/2467301041/Design+Tokens+and+How+We+Name+Them?focusedCommentId=2469500459#comment-2469500459)

Reply:
interesting edge case here for this part of the naming system for us to consider: https://github.com/okta/odyssey/pull/1300#discussion_r796515594

Reply:
FWIW I updated my PRs to follow the `TextColor` convention just so we can keep moving forward with a unified API for now